engines/scumm/input.cpp

```diff
@@ -897,20 +898,26 @@ void ScummEngine::processKeyboard(Common::KeyState lastKeyHit) {
 		char sliderString[256];
 		PauseToken pt;
 
-		if ((VAR_PAUSE_KEY != 0xFF && (lastKeyHit.ascii == VAR(VAR_PAUSE_KEY))) ||
-			(lastKeyHit.keycode == Common::KEYCODE_SPACE && _game.features & GF_DEMO)) {
-			// Force the cursor OFF...
-			int8 oldCursorState = _cursor.state;
-			_cursor.state = 0;
-			CursorMan.showMouse(_cursor.state > 0);
-			// "Game Paused.  Press SPACE to Continue."
-			if (_game.version > 4)
-				showBannerAndPause(0, -1, getGUIString(gsPause));
-			else
-				showOldStyleBannerAndPause(getGUIString(gsPause), 12, -1);
-
-			_cursor.state = oldCursorState;
-			return;
+		if (((VAR_PAUSE_KEY != 0xFF && (lastKeyHit.ascii == VAR(VAR_PAUSE_KEY))) ||
```

Comment:
I wonder why did you add two extra sets of parentheses here. Looks unnecessary to me.

Reply:
A result of me coding at night :-P thanks!